packages/api-plugin-authentication/src/config.js

```diff
@@ -1,10 +1,13 @@
 import envalid from "envalid";
 
-const { str } = envalid;
+const { str, bool } = envalid;
 
 export default envalid.cleanEnv(
   process.env,
   {
+    ACCOUNTS_JS_RETURN_TOKENS_AFTER_RESET_PASSWORD: bool({ default: false }),
+    ACCOUNTS_JS_ACCESS_TOKEN_EXPIRES_IN: str({ default: "90m" }),
+    ACCOUNTS_JS_REFRESH_TOKEN_EXPIRES_IN: str({ default: "30d" }),
```

Comment:
We are providing 30d as default for refresh-token, but the default in accountsjs is 7d [link](https://github.com/accounts-js/accounts/blob/ff983b8f9603f6295f2b7c9830a7926bc36e81c4/packages/server/src/accounts-server.ts#L51)